common/templates/context_funcs.go

```diff
@@ -1335,3 +1336,154 @@ func (c *Context) tmplEditNickname(Nickname string) (string, error) {
 
 	return "", nil
 }
+
+func (c *Context) tmplSort(slice []interface{}, sortargs ...interface{}) (interface{}, error) {
+	if c.IncreaseCheckCallCounterPremium("sortfuncs", 1, 3) {
+		return "", ErrTooManyCalls
+	}
+
+	var dict SDict
+	var err error
+
+	// We have optional args to set the output of the func
+	//
+	// Reverse
+	// Reverses the order
+	// From [0 1 2] to [2 1 0]
+	//
+	// Subslices
+	// By default the function returns a single slice with all the values sorted
+	// setting subslices to true will make the function return a set of sublices
+	// based on the input type/kind
+	// From [1 2 3 a b c] to [[1 2 3] [a b c]]
+	//
+	// Emptyslices
+	// By default the function only returns the slices that had an input to them
+	// if you sort only strings, the output would be slice of strings.
+	// But with this flag the function returns all possible slices, this is helpful for indexing
+	// From [[1 2 3] [] [a b c] [map[a:1 b:2]]] to [[1 2 3] [] [a b c] [] [] [map[a:1 b:2]] []]
+	//
+	// We can have up to 7 subslices total:
+	// intSlice, floatSlice, stringSlice, timeSlice, sliceSlice, mapSlice and defaultSlice
+	switch len(sortargs) {
+	case 0:
+		input := make(map[string]interface{}, 3)
+		input["reverse"] = false
+		input["subslices"] = false
+		input["emptyslices"] = false
+
+		dict, err = StringKeyDictionary(input)
+		if err != nil {
+			return "", err
+		}
```

Comment:
This could be simplified to this, right? (not tested)
```suggestion
		dict = SDict{
			"reverse": false,
			"subslices": false,
			"emptyslices": false,
		}
```

Reply:
Yup, you are right